src/utils/constants.ts

```diff
@@ -190,6 +190,11 @@ ZDFieldValidation[ZDFieldTypes.Regex] = {
     RegexError: 'Regex field not valid',
 };
 
+export const ZDRoles = {
+    admin: 'admin',
+    agent: 'agent',
+};
```

Comment:
Maybe use an enum here

```suggestion
export enum ZDRoles {
    admin = 'admin',
    agent = 'agent',
};
```

Reply:
I have a ticket for using enums, but want to spend time understanding the benefits and tradeoffs first

Reply:
added into https://mattermost.atlassian.net/browse/MM-33764

Reply:
The only tradeoff I can think of is that it wouldn't be usable in a javascript file, but since the whole project is typescript we are good on that front.